apptools/io/h5/dict_node.py

```diff
@@ -69,7 +69,7 @@ def __contains__(self, key):
         return key in self.data
 
     def keys(self):
-        return self.data.keys()
+        return list(self.data.keys())
```

Comment:
This change needs discussion.

If we don't make this change, we have different behavior on Python 2 and Python 3.
If wrap the call with `list`, the return value of `H5DictNode.keys` differs from how a normal dictionary behaves.

Reply:
I think that because this method is part of the dictionary interface then it should act like a normal dictionary. Remove the `list` wrapper.